src/common/SurgeStorage.cpp

```diff
@@ -2269,6 +2269,36 @@ bool SurgeStorage::resetToCurrentScaleAndMapping()
     return true;
 }
 
+void SurgeStorage::loadTuningFromSCL(const fs::path &p)
+{
+    try
+    {
+        retuneToScale(Tunings::readSCLFile({p.u8string()}));
+    }
+    catch (Tunings::TuningError &e)
+    {
+        retuneTo12TETScaleC261Mapping();
+        reportError(e.what(), "SCL Error");
+    }
+    if (onTuningChanged)
+        onTuningChanged();
+}
+
+void SurgeStorage::loadMappingFromKBM(const fs::path &p)
+{
+    try
+    {
+        remapToKeyboard(Tunings::readKBMFile({p.u8string()}));
+    }
+    catch (Tunings::TuningError &e)
+    {
+        remapToConcertCKeyboard();
```

Comment:
Why does the above remap to 12TET on error, and this one to ConcertC? Should probably be consistent.

Reply:
This one is a keyboard mapping (.kbm) which is distinct from a plain tuning (.scl). Its default is 'ConcertCKeyboard()'.